src/app.js

```diff
@@ -3,11 +3,184 @@ import $ from 'jquery';
 import _ from 'underscore';
 
 // CSS
-import './css/foundation.css';
-import './css/style.css';
+//import './css/foundation.css';
+//import './css/style.css';
+
+// Models and Collections
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
+import Reservation from './app/models/reservation';
+
+// Vars that need to be defined up front
+let tripRowTemplate;
+let tripDetailsTemplate;
+let trip = new Trip();
```

Comment:
I don't think this `trip` variable declared at "module scope" (at the top level of the file) is needed.

It's only being used for line 32 in `getTrip()`, and in that case a local variable within the function should be sufficient.